inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile

```
@@ -21,8 +21,8 @@ FROM gcc:5.4.0 as protobuf
 RUN apt-get update && apt-get install -y unzip \
     && rm -rf /var/lib/apt/lists/
 ARG PROTOBUF_VERSION=2.5.0
-RUN wget -O "/tmp/protobuf-${PROTOBUF_VERSION}.zip" "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
-    && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
+RUN curl -LOk "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
+    && unzip "protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
     && cd /opt/protobuf-${PROTOBUF_VERSION} \
     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
     && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"
```

It seems that the zip file is download, but not deleted after unzipping.

```suggestion
RUN curl -Lk "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
    -o "/tmp/protobuf-${PROTOBUF_VERSION}.zip"
    && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
    && cd /opt/protobuf-${PROTOBUF_VERSION} \
    && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
    && rm -rf "/tmp/*"
```